runtimes/eden/src/constants.rs

```diff
@@ -143,4 +143,9 @@ mod tests {
 		assert_eq!(deposit(0, 1000), 60000000000 as Balance);
 		assert_eq!(deposit(0xFFFF_FFFF, 0xFFFF_FFFF), 901943131950000000 as Balance);
 	}
+
+	#[test]
+	fn polkadot_constants() {
+		assert_eq!(100 * POLKADOT_CENT, POLKADOT_EXISTENTIAL_DEPOSIT);
```

Comment:
A better improvement here is that instead of redefining Polkadot constants here, we use them from the polkdot-sdk directly.

Reply:
https://linear.app/nodle/issue/CHA-453